ui/src/app/applications/components/application-create-panel/source-panel.tsx

```diff
@@ -0,0 +1,285 @@
+import {AutocompleteField, DataLoader, DropDownMenu} from 'argo-ui';
```

Comment:
Did you take this source from https://github.com/argoproj/argo-cd/blob/8f0d3d0f6ad5b0cc94c704ec2fd2c26fa97d8202/ui/src/app/applications/components/application-parameters/source-panel.tsx#L12 ? The next step is to common this up.